temp-rh-pwr-control/temp-rh-pwr-control.ino

```diff
@@ -242,13 +275,18 @@ void initRelay() {
   digitalWrite(RELAYPIN, RELAY_ON);
   isPowerOn = true;
   isAutoPowerOn = true;
+  isLampOn = isPowerOn;
```

Comment:
Is there possibility that lamp status will differ from power status? If no, there is no need in isLampOn variable